### ISSUE TITLE
addMessageProp does not work

### ISSUE THREAD

**Issue by Samurais** [superscriptjs/superscript#301]
Add a dialog as below
```
+ __preview
- {@__preview_question_kickoff}

+ __preview_question_kickoff
- Do you want to play word games? Yes? ^addMessageProp(voice_message_only,chatbot1p)
- Let's play word games OK? ^addMessageProp(voice_message_only,chatbot2p)
```
## Expected Behavior
It should have voice_message_only in reply data.

## Current Behavior
The function does run, but the prop is not attached. 

## Possible Solution

```
+ __preview
- {@__preview_question_kickoff} ^addMessageProp(voice_message_only,chatbot1p)

+ __preview_question_kickoff
- Do you want to play word games? Yes? 
- Let's play word games OK? 
```

Above script works as expected, but the much more practical case is to enable attach different value for each reply in grouped replies. 

## Context
Fulfill a requirement to send voice message to client.

## Your Environment
* Version used: Node.js v6.9.1
* Environment name and version: run against https://github.com/superscriptjs/superscript/commit/82e46877931f28e75eec63b034209c877b7f1f5b
* Server type and version: 
* Operating System and version: mac osx

**Comment by nsaraiya** [comment]
I tried your original example but the system seems to work as expected. Am I missing anything here?
ss file:

[10:56]  
> topic:keep main

   + ~emohello *~2
   - Hi!

   + __preview
   - {@__preview_question_kickoff}

   + __preview_question_kickoff
   - Do you want to play word games? Yes? ^addMessageProp2(voice_message_only,chatbot1p)
   - Let's play word games OK? ^addMessageProp2(voice_message_only,chatbot2p)

< topic

[10:57]  
Response:

[10:57]  
You> __preview

Bot> Let's play word games OK? addMessageProp received: voice_message_only chatbot2p
You> __preview

Bot> Let's play word games OK? addMessageProp received: voice_message_only chatbot2p
You> __preview

Bot> Do you want to play word games? Yes? addMessageProp received: voice_message_only chatbot1p
You>

**Comment by Samurais** [comment]
I can re-produce the problem in this repo:
(https://github.com/Samurais/superscript-sample](https://github.com/Samurais/superscript-sample)

Please take a look, I am very appreciated your help.

**Comment by benhjames** [comment]
Fixed in v1.0.0 - cheers!
